print_list.js

```diff
@@ -0,0 +1,34 @@
+
+class Node {
+  constructor(value, next) {
+    this.value = value;
+    this.next = next;
+  }
+
+  getNext() {
```

Comment:
I will probably recommend the accessor descriptors get and set. 
```suggestion
  get next() {
       return this.next
  }
```